### PR TITLE
im highj

### PR DESCRIPTION
**Pull request by YOUWILLDIE666**
accept this now

### CODE REVIEW

**Comment by YOUWILLDIE666** [comment]
merge*

**Comment by YOUWILLDIE666** [comment]
BRO I ACCIDENTALLY

File: mainv2.cpp [hunk 1]
```diff
@@ -0,0 +1,86 @@
+#include <windows.h>
+#include <tlhelp32.h>
+#include <iostream>
+#include <fstream>
+#include <string>
+
+DWORD asdasfv(const std::wstring& processName) {
```
Comment (173205081) [inline mainv2.cpp:7]: why

File: mainv2.cpp [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+bool isInteresting(HANDLE hProcess, std::uint32_t offset) {
+    BYTE buffer[4];
+    SIZE_T bytesRead;
+    
+    if (ReadProcessMemory(hProcess, (LPCVOID)offset, &buffer, sizeof(buffer), &bytesRead) && bytesRead == sizeof(buffer)) {
+        std::uint32_t value = *reinterpret_cast<std::uint32_t*>(buffer);
+
+        // actually a placeholder
+        if (value != 0 && value < 1000000) { // replace this please
```
Comment (173205081) [inline mainv2.cpp:35]: you could've implemented something here

File: mainv2.cpp [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+void log(DWORD processID) {
+    HANDLE hProcess = OpenProcess(PROCESS_VM_READ | PROCESS_QUERY_INFORMATION, FALSE, processID);
+    if (hProcess == NULL) {
+        std::cerr << "Failed to open process." << std::endl;
+        return;
+    }
+
+    std::ofstream logFile("offsets_log"+(const char*)processID+".c", std::ios::out);
```
Comment (173205081) [inline mainv2.cpp:49]: ????

File: mainv2.cpp [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+        logFile << "}\n";
+        logFile.close();
+    }
+
+    CloseHandle(hProcess);
+}
+
+int main() {
+    std::wstring processName = L"ihavenoideaifthisevenworks.exe"; // replace with the target application
```
Comment (173205081) [inline mainv2.cpp:77]: ok

**Review by 173205081 — APPROVED** [review]
Fine

**Pull request merged**